package/Dockerfile

```diff
@@ -43,6 +43,9 @@ ENV CATTLE_SYSTEM_AGENT_UPGRADE_IMAGE rancher/system-agent:${CATTLE_SYSTEM_AGENT
 # System charts minimal version
 ENV CATTLE_FLEET_MIN_VERSION=100.0.0+up0.3.6
 ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=100.0.0+up0.2.0-alpha01
+ENV CATTLE_AKS_OPERATOR_MIN_VERSION=100.0.0+1.0.100
+ENV CATTLE_EKS_OPERATOR_MIN_VERSION=100.0.0+1.1.100
+ENV CATTLE_GKE_OPERATOR_MIN_VERSION=100.0.0+1.1.100
```

Comment:
These haven't been reverted yet in charts repo, but they'll stay at 2.0.0 etc...

Reply:
Oh I see, this can be modified as long as we make a decision on versioning...

Reply:
@StrongMonkey can we remove the min versions so they are always updated?